.changeset/lemon-bulldogs-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/blocks-admin": minor
+---
+
+feature: add `maxVisibleBlocks` prop to `createBlocksBlock`
```

Comment:
```suggestion
Add `maxVisibleBlocks` option to `createBlocksBlock`
```

- A minor release indicates a feature
- Prop is somewhat reserved for React props, so option is better